package/yast2-tune.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Dec 17 15:13:33 UTC 2020 - Steffen Winterfeldt <snwint@suse.com>
+
+- add I/O device autoconfig checkbox on s390 (bsc#1168036)
+- 4.2.4
```

Comment:
Please, bump version to `4.3.0`. We are supposed to use `4.3.x` for `master` at this point.